src/ElvTenant.js

```diff
@@ -802,6 +802,106 @@ class ElvTenant {
 
     return res;
   }
+
+  async TenantFundUser({ asUrl, tenantId, userAddress, amount = 0.2}) {
+
+    console.log("TenantFundUser");
+    console.log(`as_url: ${asUrl}`);
+    console.log(`tenant_id: ${tenantId}`);
+    console.log(`user_address: ${userAddress}`);
+    console.log(`amount: ${amount}`);
+
+    // Initialize configuration
+    const config = {
+      configUrl: Config.networks[Config.net],
+      mainObjectId: Config.mainObjects[Config.net],
+    };
+
+    // Initialize EluvioLive and ElvAccount
+    const eluvioLive = new EluvioLive(config);
+    await eluvioLive.Init({
+      debugLogging: this.debug,
+      asUrl
+    });
+
+    const elvAccount = new ElvAccount({
+      configUrl: Config.networks[Config.net],
+      debugLogging: this.debug,
+    });
+    await elvAccount.Init({ privateKey: process.env.PRIVATE_KEY });
+
+    // check valid user address
+    if (!Utils.ValidAddress(userAddress)) {
+      throw Error(`Invalid user address provided: ${userAddress}`);
+    }
+    const usrAddr = Utils.FormatAddress(userAddress);
+
+    // check tenant root key
+    const tenantAddr = Utils.HashToAddress(tenantId);
+    if (!Utils.ValidAddress(tenantAddr)) {
+      throw Error(`Invalid tenantId provided: ${tenantId}`);
+    }
+    const abi = fs.readFileSync(
+      path.resolve(__dirname, "../contracts/v3/BaseTenantSpace.abi")
```

Comment:
not your doing at all obviously, but, if we're going to commit anything, let's take an opportunity to avoid hard-coding this and duplicating the code in some many spots.  (as is this is just too hard to change, eg, if this became a indexer query at some point.)

let's abstract this out into a call that's shared, allowing us to reduce the number of hard-coded refs to this file and the specific contact call args, etc.

unless there is a huge rush on this..